src/net.h

```diff
@@ -32,6 +32,7 @@
 
 #ifndef WIN32
 #include <arpa/inet.h>
+#include <poll.h>
```

Comment:
Should this be behind `#ifdef USE_POLL` instead? To match the `pollfd` reference below.